examples/GMLS_DivergenceFree.cpp

```diff
@@ -176,6 +176,10 @@ bool all_passed = true;
     // need Kokkos View storing true solution
     Kokkos::View<double**, Kokkos::DefaultExecutionSpace> vector_sampling_data_device("samples of true vector solution",
             source_coords_device.extent(0), dimension);
+    Kokkos::View<double**, Kokkos::DefaultExecutionSpace> curl_vector_sampling_data_device("samples of curl of true vector solution",
+            source_coords_device.extent(0), dimension);
+    Kokkos::View<double**, Kokkos::DefaultExecutionSpace> curlcurl_vector_sampling_data_device("samples of curl of true vector solution",
```

Comment:
samples of curl-curl?

Reply:
yeah sorry it should read that - can fix it real quick